testCourse/questions/barcodeSubmission/question.html

```diff
@@ -0,0 +1,3 @@
+<h6 style="padding-bottom: 20px;">Enter the barcode from the printed scrap paper sheet given to you by your instructor to submit your work:</h6>
```

Comment:
Nit: for the sake of accessibility, this is not a heading.

```suggestion
<p>Enter the barcode from the printed scrap paper sheet given to you by your instructor to submit your work:</p>
```